### PR TITLE
Add 9c-stake-test application in 9c-internal

### PR DESCRIPTION
**Pull request by XxshiftxX**

### CODE REVIEW

File: 9c-internal/9c-stake-test/values.yaml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    node.kubernetes.io/instance-type: m5d.xlarge
+
+validator:
+  count: 1
+  image:
+    repository: planetariumhq/ninechronicles-headless
+    pullPolicy: Always
+
+  consensusSeedStrings:
+  - 02101677a587605b9c900ab15241844afa99ae05e952ad1b9d2680bdf3b029deb9,tcp-seed-1,31235
```
Comment (moreal) [inline 9c-internal/9c-stake-test/values.yaml:57]: Is it okay? 🤔

File: 9c-internal/9c-stake-test/values.yaml [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    size: 1Gi
+
+  nodeSelector:
+    node.kubernetes.io/instance-type: m5d.large
+
+  env:
+  - name: DOTNET_gcServer
+    value: "1"
+  - name: RpcConfig__Host
+    value: 9c-internal-rpc-1.nine-chronicles.com
```
Comment (moreal) [inline 9c-internal/9c-stake-test/values.yaml:166]: This should be updated.

File: 9c-internal/9c-stake-test/values.yaml [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  downloadSnapshot: false
+  slackChannel: "9c-internal"
+  image: "planetariumhq/ninechronicles-snapshot:git-dc31bc2b777d9ca18d101bd1ab81351cce1b32c4"
+  path: internal
+
+# if you want to delete PVC with the volume provisioned together, set this value "Delete"
+volumeReclaimPolicy: "Retain"
+
+seed:
+  count: 0
```
Comment (sky1045) [inline 9c-internal/9c-stake-test/values.yaml:37]: is it okay without any seed node?

Comment (XxshiftxX) [reply 9c-internal/9c-stake-test/values.yaml:37]: I thought it'll be ok cause it starts from empty genesis block.

File: 9c-internal/9c-stake-test/values.yaml [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+  peerStrings:
+  - "033369e95dbfd970dd9a7b4df31dcf5004d7cfd63289d26cc42bbdd01e25675b6f,9c-internal-stake-validator.nine-chronicles.com,31234"
+  iceServers:
+  - "turn://0ed3e48007413e7c2e638f13ddd75ad272c6c507e081bd76a75e4b7adc86c9af:0apejou+ycZFfwtREeXFKdfLj2gCclKzz5ZJ49Cmy6I=@turn-us.planetarium.dev:3478"
+
+  networkType: Internal
+  consensusType: pbft
+
+  resetSnapshot: false
+  useExternalSecret: true
```
Comment (sky1045) [inline 9c-internal/9c-stake-test/values.yaml:25]: we need to turn off this option to use other private keys

**Comment by moreal** [comment]
Since this test ends, I close this pull request 👀

**Pull request closed without merging**